src/integrations/GoogleAds/browser.js

```diff
@@ -175,6 +172,36 @@ class GoogleAds {
     return conversionData;
   }
 
+  sendEvent(type, eventName) {
```

Comment:
```suggestion
  shouldSendEvent(type, eventName) {
```
Function name is not giving its full intent, also check if you can reduce the cognitive complexity of this function  there are too many nested conditions which makes it difficult to read

Reply:
updated